### ISSUE TITLE
Validator GUI not connecting at ```localhost:8080```

### ISSUE THREAD

**Issue by chris-metz** [Shardeum/shardeum-bug-reporting#33]
### What is the issue?

Having setup a validator node via the docs at https://docs.shardeum.org/node/run/validator, I get an empty response when trying to connect to the GUI with a browser or curl.
    
### What impact does the issue have?

Not being able to run a Validator Node.

### How to reproduce the issue? 

The docker image is successfully running and the logs last output of the running container is:

```
shardeum-dashboard  | Getting CA Private Key
shardeum-dashboard  | y
shardeum-dashboard  | Starting operator gui
shardeum-dashboard  | done
```

running **operator-cli gui status** says it is running:

```
node@8eeb231b367f:~/app$ operator-cli gui status
name: operator-gui
uptimeInSeconds: 929.628
restarts: 1
status: online
cpuUsagePercent: 0.3
memUsedInBytes: 58351616
link: http://localhost:8080/
```

However, when trying to acces via browser it says EMPTY_RESPONSE. The same happens even when trying to curl localhost from within the container:

```
node@8eeb231b367f:~/app$ curl localhost:8080
curl: (52) Empty reply from server
```

Could I be missing something here?

**Comment by MarcusWentz** [comment]
1. What operating system are you running for this setup?

2. Is this on a local or remote computer? 

3. What do you see when you run 

```shell
docker --version
```

?

4. What do you see when you run 

```shell
docker-compose --version
```

?

**Comment by MarcusWentz** [comment]
What do you see in your web browser with:

```
https://localhost:8080/
```

and

```
http://localhost:8080/
```

?

**Comment by xhzi** [comment]
Facing similar issue
```
node@083f86a4d797:~/app$ curl localhost:8080
curl: (7) Failed to connect to localhost port 8080: Connection refused
```
status of gui is running
```
node@083f86a4d797:~/app$ operator-cli gui status
name: operator-gui
uptimeInSeconds: 1.706
restarts: 35
status: online
cpuUsagePercent: 0
memUsedInBytes: 129904640
link: http://localhost:8080/
```

**Comment by sintez70** [comment]
I was helped by running through
https://YOURIPVPS:8080/maintenance

**Comment by chris-metz** [comment]
OK, nevermind. I mistakenly tried with http instead of https, which caused the browser to cancel the connection because of wrong protocol.

When connecting with **https** I can sucessfully connect.
Thanks!

**Comment by mesahin001** [comment]
But other than standart port (8080) still not working. Yestarday I have selected at the installation i.e port to 8085 and after that portal was not accessible neither 8080 nor 8085. I installed it again with port 8080 and that was OK.
I played after again with the port and changed the parameter in .env from 8080 to 8081 and docker restarted. Again it was not accessible from both ports (8080, 8081).
At the end I have changed ports in docker-compose.yml again with another port and the result was same: not accessible.

I think the port 8080 is somewhere hard wired.
